dbt_semantic_interfaces/protocols/semantic_manifest.py

```diff
@@ -23,5 +24,10 @@ def metrics(self) -> Sequence[Metric]:  # noqa: D
     def interfaces_version(self) -> str:  # noqa: D
```

Comment:
Is this now duplicate information? Should we remove this property in favor of `dsi_package_version` in `project_configurations`?